### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by samhstn**
Merge after #156 

- [x] Adds documentation generating instructions to the readme
- [x] Updates deps with documentation module

### CODE REVIEW

**Review by jackcarlisle — COMMENTED** [review]
Reviewing now!

**Review by jackcarlisle — CHANGES_REQUESTED** [review]
Hey Sam, could you rebase with master from the last PR so that I can see which additions are specific to this PR?

**Comment by samhstn** [comment]
Sure thing 👍

File: README.md [hunk 1]
```diff
@@ -62,3 +62,14 @@ at the root of your project
 ## Routes
 
 Check the application routes with `mix pheonix.routes`
+
+## Further Documentation
+
+You can view further technical documentation by running:
+```bash
```
Comment (iteles) [inline README.md:69]: Didn't know about this yet, pretty cool!

**Review by iteles — APPROVED** [review]
Adding to the docs is *always* a great idea 👍

**Comment by samhstn** [comment]
@jackcarlisle Rebased, more documentation will be made in a separate pr

**Review by jackcarlisle — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (samhstn) [reply README.md:69]: Thanks @iteles, I think @jackcarlisle and I originally saw this on [this udemy tutorial](https://www.udemy.com/the-complete-elixir-and-phoenix-bootcamp-and-tutorial/) and I found [this](https://brainlid.org/elixir/2016/09/01/phoenix-project-documentation.html) really useful for an introduction on how to document in a phoenix project.

I'm going to play with it over the next few days in this project, then maybe write up my findings in [dwyl/learn-phoenix-framework](https://github.com/dwyl/learn-phoenix-framework)

**Pull request merged**